repos/spack_repo/builtin/packages/quda/package.py

```diff
@@ -57,7 +57,7 @@ class Quda(CMakePackage, CudaPackage, ROCmPackage):
 
     variant("mpi", default=False, description="Enable MPI support")
     variant("qmp", default=False, description="Enable QMP")
-    variant("qio", default=False, description="Enable QIO")
+    variant("qio", default=False, description="Enable QIO (require QMP)", when="+qmp")
```

Comment:
```suggestion
    variant("qio", default=False, description="Enable QIO", when="+qmp")
```
Nitpick: the requirement is already documented by the variant, so this feels a bit redundant. No worries if you want to leave it in.